src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    private Ball ball = new Ball();
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public String getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
you should return a ball from this method, not a string